save-demo/src/main/kotlin/com/saveourtool/save/demo/controller/ManagementApiController.kt

```diff
@@ -35,16 +36,11 @@ class ManagementApiController(
     fun getDemoStatus(
         @PathVariable organizationName: String,
         @PathVariable projectName: String,
-    ): Mono<DemoStatus> = demoService.findBySaveourtoolProjectOrNotFound(organizationName, projectName) {
-        "Could not find demo for $organizationName/$projectName."
+    ): Mono<DemoStatus> = blockingToMono {
+        demoService.findBySaveourtoolProject(organizationName, projectName)
     }
-        .map {
-            /*
-             * todo:
-             * kubernetesService.getStatus(it)
-             */
-            DemoStatus.STOPPED
-        }
+        .flatMap { deferredToMono { kubernetesService.getStatus(it) } }
```

Comment:
suggestion to look into https://github.com/Kotlin/kotlinx.coroutines/blob/master/reactive/kotlinx-coroutines-reactor/README.md and re-use it (if we decided to you both ProjectReactor and suspend functions)

Reply:
**P. S.** Please also note, that `mono()` and `flux()` functions accept a coroutine context as the 1st argument, and a `Scheduler` from reactor-core can be converted to a context/dispatcher.

Reply:
Will override existing `deferredToMono` with `kotlinx-coroutines-reactor`

Reply:
Done